src/xcode/ENA/ENA/Source/Models/Exposure/ExposureManager.swift

```diff
@@ -219,15 +220,27 @@ final class ENAExposureManager: NSObject, ExposureManager {
 	/// Activates `ENManager`
 	/// Needs to be called before `ExposureManager.enable()`
 	func activate(completion: @escaping CompletionHandler) {
-		Log.info("Trying to activate ENManager.")
```

Comment:
Hm the old log was more correct, wasn't it?
We activate the ENManager, which will then activate ENF right?

Reply:
I can revert it.